src/derive/hashtocurve.rs

```diff
@@ -0,0 +1,90 @@
+// This module implements "try and increment" hashing to short Weierstrass curves
+// hash_to_field is from: https://github.com/zcash/pasta_curves
+
+use crate::CurveExt;
+use ff::{FromUniformBytes, PrimeField};
+use static_assertions::const_assert;
+
+// TODO: use simplified swu algorithm
```

Comment:
So to use simplified swu method, we need to find the isogeny parameters right? Is there a deterministic script to do that? (this one might be it https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/blob/main/poc/iso_values.sage?)

Reply:
Yes, we need first convert A=0 case to A*B != 0 curve before using simplified swu. en... Let me take a look at this code.

Reply:
I have a complete script to find Z and the SWU isogenies here as well: https://github.com/mratsim/constantine/blob/d996ccd/sage/derive_hash_to_curve.sage#L145-L275